.github/workflows/release.yml

```diff
@@ -20,10 +20,50 @@ on:
   workflow_dispatch:
```

Comment:
how about running this when pushing a tag? would be a workaround to avoid the SBOM commits. I mean we could still create the tag and choose it than in the release, but this additional step could be avoided

Reply:
Good idea, will implement that.